ofl/cairoplay/METADATA.pb

```diff
@@ -30,5 +29,6 @@ axes {
 }
 source {
   repository_url: "https://github.com/Gue3bara/Cairo"
-  commit: "78a8a4def71f06a4460fc5807777acd1672d51cb"
+  commit: "76d942d5bcc0ea491b6ff7d9d199bd0bb817f127"
 }
+primary_script: "Arab"
```

Comment:
Was this line added at the end by packager? I wonder if it should follow the former order (after subset)

Reply:
Yes. I didn't do anything

Reply:
The packager didn't add this. We added the primary_script to these Arabic color fonts because the previews on fonts.google.com were showing Latin only.

When you run the packager, it will keep the primary_script field if one has been included. The order cannot be changed because new fields have to be appended to the end of the proto, https://github.com/googlefonts/gftools/blob/main/Lib/gftools/fonts_public.proto#L31

Reply:
Thanks good to know!